Vagrantfile

```diff
@@ -1,9 +1,20 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+unless Vagrant.has_plugin?("vagrant-service-manager")
+  raise <<-MSG
```

Comment:
Can you examine this code relative to this: https://github.com/projectatomic/adb-atomic-developer-bundle/issues/245

Can we do a cleaner error and exit?